docs/explanations/domain_models_of_synapse.md

```diff
@@ -0,0 +1,64 @@
+# Domain Model of Synapse
+Here you'll find more information about how all of the models/concepts of Synapse
+work together and what their purpose is.
+
+## Entity
+Any distinct item in Synapse that has its own synID, including a Project, Folder, File, Wiki, Dataset, and View, among other items.
+
+Synapse entities are identified by a unique identifier called a Synapse ID, or synID. The synID is represented by the prefix “syn” followed by numbers (for example, syn12345678). This identifier can be used to refer to a specific entity on the web and through the programmatic clients.
+
+## [Projects](../tutorials/python/project.md)
+Projects in Synapse are “containers” that group relevant content and people together. All data must be uploaded into a project. Projects can be private so only you can see the contents, they can be shared with your collaborators, or they can be made public so anyone on the web can view your research.
```

Comment:
```suggestion
Projects in Synapse are “containers” that group relevant content and people together. All data must be uploaded into a Project. Projects can be private so only you can see the contents, they can be shared with your collaborators, or they can be made public so anyone on the web can view your research.
```

Reply:
I went the other way and lowercased things for consistency